tests/Integration/inc/Engine/Optimization/DelayJs/Admin/Subscriber/setOptionOnUpdate.php

```diff
@@ -1,45 +1,33 @@
 <?php
 
-namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DelayJs\Admin\Subscriber;
+namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization\DelayJS\Admin\Subscriber;
 
-use Mockery;
-use WP_Rocket\Admin\Options_Data;
-use WP_Rocket\Engine\Optimization\DelayJS\Admin\Settings;
-use WP_Rocket\Engine\Optimization\DelayJS\Admin\Subscriber;
-use WP_Rocket\ServiceProvider\Options;
 use WP_Rocket\Tests\Integration\TestCase;
-use Brain\Monkey\Functions;
 
 /**
  * @covers \WP_Rocket\Engine\Optimization\DelayJS\Admin\Subscriber::set_option_on_update
  *
- * @group  DelayJs
+ * @group  DelayJS
  */
 class Test_SetOptionOnUpdate extends TestCase{
-
-	public function tearDown() {
-		parent::tearDown();
-
-		$options = new \WP_Rocket\Admin\Options( 'wp_rocket_' );
-		$options->set( 'settings', null );
-	}
-
 	/**
 	 * @dataProvider configTestData
 	 */
-	public function testShouldDoExpected( $old_version, $valid_version ){
-
+	public function testShouldDoExpected( $old_version, $valid_version ) {
 		do_action( 'wp_rocket_upgrade', '', $old_version );
 
-		$options      = new \WP_Rocket\Admin\Options( 'wp_rocket_' );
-		$option_array = new Options_Data( $options->get( 'settings' ) );
+		$options = get_option( 'wp_rocket_settings' );
 
 		if ( $valid_version ) {
-			$this->assertEquals( $option_array->get('delay_js', null), 0 );
+			$this->assertSame(
+				0,
+				$options['delay_js']
+			);
 		} else {
-			$this->assertNull( $option_array->get('delay_js', null) );
+			$this->assertSame(
+				1,
+				$options['delay_js']
+			);
```

Comment:
@engahmeds3ed I updated this after updating the fixture to use the `SettingsTrait`, to check for the value of the option key which should always be set